README.rst

```diff
@@ -5,6 +5,20 @@ Pytorch Datastream
 .. image:: https://badge.fury.io/py/pytorch-datastream.svg
        :target: https://badge.fury.io/py/pytorch-datastream
 
+.. image:: https://img.shields.io/github/tag/Aiwizo/pytorch-datastream.svg
```

Comment:
I don't like the tag badge and it's also orange which is an alarming color :)

Reply:
I didn't take you for a color sensitive person :D but yes, now when you say it yes it looks like some build or something went half bad
Also with the current setup I think version and tag will never be different so might as well remove tag